get-to-know-security-manager/settings-screen.md

```diff
@@ -89,5 +90,16 @@ You can choose from two types of schedules:
 
 #### **Date & Time**
 
-By default, the report is scheduled on 8 AM, weekly, recurring on Monday. However you can adjust it to the time you prefer. You can choose how frequent will the snapshot be performed and depending on the frequency you have selected \(weekly or monthly\), you will also be able to choose on which day will the snapshot be recurring.
+By default, the report is scheduled on 8 AM, weekly, recurring on Monday. However, you can adjust it to the time you prefer. You can choose how frequent will the snapshot be performed and depending on the frequency you have selected \(weekly or monthly\), you will also be able to choose on which day will the snapshot be recurring.
 
+## **Audit**
+
+### **Audit Logs**
+
+By enabling the collection of Office 365 audit logs, you can find out about the activities of admins and users within your organization. Also, here you can change the location of the folder for collection of audit logs on your computer. If you want to check and edit which activities are going to be collected, click on the link "Select activities to collect".
+
+## **Data Retention**
+
+### **Data retention Policy**
+
+With this option, you can allow Syskit Security Manager to automatically delete an old data from its storage. You can allow the application to remove snapshots that are older than a certain number of days, weeks or months. There is also the same option for removing old audit logs data. For audit logs, you must enter your email address in order to be alerted when Syskit Security Manager will stop collecting data if disk usage reaches a critical limit above 95%.
```

Comment:
više o tome kako rade ovi warninzi i općenito data retention vidi ovdje: https://acceleratio.visualstudio.com/SysKitSecurityManager/_workitems/edit/40733
ovaj dio isto razlomiti + detaljnije opisati kako točno ovo radi, kada se briše itd. - opisati dovoljno jasno da svi korisnici mogu skužiti o čemu je riječ, ali opet ne ulaziti previše u detalje